src/utilities/manifestStabilityUtils.ts

```diff
@@ -85,8 +100,8 @@ export async function checkPodStatus(
    for (let i = 0; i < iterations; i++) {
       await sleep(sleepTimeout)
 
-      core.debug(`Polling for pod status: ${podName}`)
-      podStatus = await getPodStatus(kubectl, podName)
+      core.debug(`Polling for pod status: ${resource.name}`)
```

Comment:
if this method is just for pods, then does this name change make it harder to understand the code?

Reply:
I think we should keep the arg as the resource type but would agree that we could make the param name `pod` or something